### DISCUSSION TITLE
PWA is not starting

### DISCUSSION THREAD

**Discussion by ashishkpaul** [vuestorefront/spree#222] [Q&A]
Hello sir, Thanks for sharing the repo. I install and compile code according the guide and hosted the project on a production mode with
 `pm2 start npm --name "StoreFront" -- run "start"` 

with a nginx ssl config file

```
upstream vue_storefront_upstream {
    server 10.0.0.123:3000;
    keepalive 64;
}
server {
    listen 443 ssl;

    server_name www.example.com;
    error_log   /var/log/nginx/example-vue-storefront-error.log;

    ssl_certificate /etc/letsencrypt/live/example.com/fullchain.pem; # managed by Certbot
    ssl_certificate_key /etc/letsencrypt/live/example.com/privkey.pem; # managed by Certbot
    include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot

location / {
        proxy_set_header  Host $host;
	proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header  X-Forwarded-Proto $scheme;
	proxy_set_header  X-Forwarded-Ssl on; # Optional
	proxy_set_header  X-Forwarded-Port $server_port;
	proxy_set_header  X-Forwarded-Host $host;

        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";

        proxy_pass http://vue_storefront_upstream;
        proxy_redirect off;
        proxy_read_timeout 240s;
    }
```
I am using spree 4.4 
can you guide me further steps to implement PWA

**Comment by rafalcymerys** [comment]
Hi @ashishkpaul - could you share some details regarding the exact issue you're experiencing?

**Reply by ashishkpaul** [reply]
yes sir, I want to install pwa app on my mobile when i run the site on mobile chrome its working good but it is not showing any download button to install the app on mobile

**Reply by ashishkpaul** [reply]
Sir i read the artical https://api.spreecommerce.org/docs/api-v2/YXBpOjMxMjQ5NjA-storefront-api-v2 where it says that we can create a mobile app through vue storefront.  Please guide me.
